script/update_rubygems_and_install_bundler

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This file was generated on 2019-01-03T20:34:23+00:00 from the rspec-dev repo.
+# DO NOT modify it by hand as your changes will get lost the next time it is generated.
+
+set -e
+source script/functions.sh
+
+if is_ruby_23_plus; then
```

Comment:
Also I see those functions but not where it is called? 🤔

Reply:
As this is rspec-rails it's `.travis.yml` is kept out of step, and I hadn't manually swapped it out, have now.

Reply:
Thanks Jon